src/main/oauth2/redirect-handler.ts

```diff
@@ -10,38 +10,31 @@
 import http from "node:http";
 import { AddressInfo } from "node:net";
 
-export class RedirectHandler {
+export default class RedirectHandler {
```

Comment:
Auth + redirecting are all in `tachyon-client` package already. Any reason not to use that?

Reply:
FYI: the redirector here is already a copy of what was implemented in tachyon-client, you can compare: https://github.com/beyond-all-reason/tachyon-client/commit/a923447faea5db786309908a85f2d0199ad4c51e, so it's not from scratch.

Overall we decided that vendoring the tachyon client code into bar lobby at the moment is a very good idea because:

1. There will be quite rapid iteration on the protocol, and needing to change tachyon-protocol package, then tachyon-client package, and only then bar-lobby seems a bit much indirection.
2. There aren't any other tachyon-client consumers planned yet